src/styles/blocks/button.css

```diff
@@ -0,0 +1,27 @@
+.buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  background-color: var(--color-blue);
+  border-radius: 5px;
+  border: 1px solid transparent;
+  list-style: none;
+
+  height: 40px;
+  width: 166px;
```

Comment:
you made `buy` class. it's a block. now you can't youse these on blocks, according to BEM